modin/pandas/series.py

```diff
@@ -70,6 +70,7 @@ class Series(SeriesCompat, BasePandasDataset):
     """
 
     _pandas_class = pandas.Series
+    __array_priority__ = pandas.Series.__array_priority__
```

Comment:
what is `__array_priority__`?

Reply:
it is used by numpy to determine when an op should return NotImplemented, e.g. `some_numpy_array.__add__(some_modin_series)`